pkgs/applications/editors/inventic/skipper.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook, makeWrapper, unzip,
+
+# Unpack libs
+boost172, curl, libX11, libpng12, libxml2, libxslt, openssl, postgresql, qt5
+, zlib }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "skipper";
+  version = "3.2.25.1701";
+  src = fetchurl {
+    url =
+      "https://downloads.skipper18.com/${version}/Skipper-${version}-Linux-all-64bit.zip";
+    sha256 = "130wfn20mj0nrp66grk7fj8xr6m4l1cp7741w40ixp8gdhih57zc";
+  };
+
+  nativeBuildInputs = [ unzip autoPatchelfHook qt5.wrapQtAppsHook makeWrapper ];
+
+  buildInputs = [
+    boost172
+    curl
+    libX11
+    libxml2
+    libxslt
+    openssl
+    postgresql
+    qt5.qtbase
+    qt5.qtscript
+    stdenv.cc.cc.lib
```

Comment:
That looks strange to me and so far I only have seen it when doing cross compile.